packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -244,9 +244,10 @@ export const connectEpic: Epic<AnyAction, AnyAction, any, any> = (action$, state
             const timeout$ = of(subscriptionError({ ...action, reason: 'WS: unsubscribe -> subscribe (unresponsive channel)' }), unsubscribe(action), subscribe(action)).pipe(
               delay(config.subscriptionUnresponsiveTTL),
               withLatestFrom(state$),
-              map(([action, state]) => ({ ...action, isActive: state.ws.subscriptions[subscriptionKey]?.active })),
+              map(([action, state]) => ({ ...action, isActive: state.ws.subscriptions[subscriptionKey]?.active || state.ws.subscriptions[subscriptionKey]?.subscribing > 0 })),
```

Comment:
I'd prefer if we were a little bit more explicit here. Maybe something like this:

```
mergeMap(([action, state]) => {
  const isActive = !!state.ws.subscriptions[subscriptionKey]?.active 
  const isSubscribing = !!(state.ws.subscriptions[subscriptionKey]?.subscribing > 0)
  return isActive || isSubscribing ? of(action) || empty()
}
```

We don't need the filter op in this case, as we made a filter ourselves.